components/app_bar/app_bar.test.tsx

```diff
@@ -134,4 +142,21 @@ describe('components/app_bar/app_bar', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should show marketplace for system admin', async () => {
```

Comment:
Could we add the negative test that doesn't show for non-system admin?

Reply:
Added